syntax = "proto2";

message request_msg1 {
	optional uint32 id = 1;
	optional string name = 2;
}